src/routes/asset.php

```diff
@@ -81,8 +81,17 @@
         if (isset($column_mapping[$params['sort']])) {
             $order_column = $column_mapping[$params['sort']];
         }
+
+        // Sorting should be reversed by default only for dates, not strings
+        $reverse = $params['sort'] === 'updated';
```

Comment:
Use `$order_column` here. Otherwise, opening the `/asset` page without parameters shows oldest assets first.